src/google/protobuf/descriptor.proto

```diff
@@ -43,6 +43,8 @@ package google.protobuf;
 option java_package = "com.google.protobuf";
 option java_outer_classname = "DescriptorProtos";
 
```

Comment:
Done. Wasn't sure whether we wanted to keep the C# options separate from the Java ones. Will check the other ones in the commit...